docs/makefile-cheat-sheet.md

```diff
@@ -46,7 +47,8 @@ In the [Cheat Sheet](#cheat-sheet), you can see which variables affect which [ta
 
 Variables are often defined before the `make` call on the same line, such as `MY_VAR=42 make my-target`. However, they can also be set on the same line _after_ the `make` call or in your environment, using, for example, `export MY_VAR=42`.
 
-**NOTE:** Targets that [meta and dependent targets](#meta-targets-and-dependent-targets) run may not all respect the same set of variables.
+!!! note
```

Comment:
is this a `!!!  tip` or `!!! note`? Or do both work?

Reply:
All the same `admonition` component, but the color and heading differ between notes and tips.

https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types